src/components/APIStarter.jsx

```diff
@@ -35,9 +390,21 @@ const APIStarter = () => {
     //     );
     //   })}
     // </ul>
-    <>{ response.length==0 ? <div></div>: (<div>
-        <QuestionView question ={response[index]} index={index} setIndex={setIndex} limit={10}/>
-    </div>)}</>
+    <>
+      {response.length == 0 ? (
+        <div></div>
```

Comment:
Same as above. Change `==` to `===` and return `null` instead of `<div></div>`

Reply:
Done